app-tasks/rf/src/rf/cli.py

```diff
@@ -6,7 +6,7 @@
 
 import click
 
-from .commands import export, process_upload
+from .commands import process_upload
```

Comment:
Not sure where to place this comment, so placing it in here, since this is related: I believe we need to removed the import of the `export` command from `app-tasks/rf/src/rf/commands/__init__.py` as well, since the `export.py` is removed, and my local attempt of `rf process-upload <id>` failed

Reply:
dc2c858